Libs/Mesh/CMakeLists.txt

```diff
@@ -70,17 +70,17 @@ install(TARGETS Mesh
 #TARGET_LINK_LIBRARIES(GenerateFidsFiles ${ITK_LIBRARIES} Mesh tinyxml)
 #INSTALL(TARGETS GenerateFidsFiles RUNTIME DESTINATION bin)
 
-ADD_EXECUTABLE(ReconstructMeanSurface ReconstructMeanSurface.cxx)
-TARGET_LINK_LIBRARIES(ReconstructMeanSurface ${ITK_LIBRARIES} tinyxml Utils Analyze)
-INSTALL(TARGETS ReconstructMeanSurface RUNTIME DESTINATION bin)
+# ADD_EXECUTABLE(ReconstructMeanSurface ReconstructMeanSurface.cxx)
```

Comment:
If they're gone let's get rid of them in the cmake lists as well.

Reply:
Removed them